Tests/MLXTests/IntegrationTests.swift

```diff
@@ -362,11 +362,11 @@ class MLXIntegrationTests: XCTestCase {
         XCTAssertEqual(result.shape, [4, 3])
         XCTAssertEqual(result.dtype, .float32)
         XCTAssertEqual(
-            result.mean().item(Float.self), -0.14204177260398865,
-            accuracy: -0.002840835452079773)
+            result.mean().item(Float.self), 0.28858500719070435,
+            accuracy: 0.005771700143814087)
         XCTAssertEqual(
-            result.sum().item(Float.self), -1.7045011520385742,
-            accuracy: -0.03409002304077149)
+            result.sum().item(Float.self), 3.463019847869873,
+            accuracy: 0.06926039695739747)
```

Comment:
Changes per https://github.com/ml-explore/mlx/pull/641 -- that changed the result of mod in some cases